### DISCUSSION TITLE
Can i use enums in tables ?

### DISCUSSION THREAD

**Discussion by ariexx** [rappasoft/laravel-livewire-tables#793] [Q&A]
hi, can i use enums datatype in livewire-tables ? because if i use Enums for my status order in database row i can't use it into livewire-tables

error is :
htmlspecialchars(): Argument #472 string) must be of type string, App\Enums\OrderStatusEnum given

![image](https://user-images.githubusercontent.com/30616892/170893044-ad966a00-a477-4004-adde-88bce1696a98.png)

**Comment by iagcs** [comment]
I just used the match function to return the correct string.

<img width="610" alt="Captura de Tela 2022-10-18 às 16 01 26" src="https://user-images.githubusercontent.com/72465191/196520764-857bbea0-6661-42c9-bba1-64fdff2c6b85.png">
